src/misc/setup.cpp

```diff
@@ -1035,8 +1035,9 @@ bool Section_prop::HandleInputline(const std::string& line)
 		}
 
 		if (p->IsDeprecated()) {
-			LOG_WARNING("CONFIG: Deprecated option '%s'", name.c_str());
-			LOG_WARNING("CONFIG: %s", p->GetHelp().c_str());
+			LOG_WARNING("CONFIG: Deprecated option '%s'\n\n%s\n",
+			            name.c_str(),
+			            p->GetHelpUtf8().c_str());
```

Comment:
Hmm, so this would make deprecation messages appear like this in the logs:

<img width="904" alt="image" src="https://github.com/user-attachments/assets/738b0b23-c729-4873-b79e-1ececfcc7b26">

Instead of this:

<img width="939" alt="image" src="https://github.com/user-attachments/assets/e415ec45-64be-4184-a94d-40828cfd5e11">

I'm on the fence about the new multiline format. What do you guys think?

Reply:
It already used to be multiline, in fact - just strangely formatted :)

Reply:
Well, previously we had multiple single-line log statements. Now we have a single multi-line log statement — that's the difference.

Reply:
I also like the new format.
The gap makes the warning stand out for easier reading.
(Plus.. warnings should be addressed anyway, so once it's solved then the log I back to line-by-line)

Reply:
Yeah, okay, let's keep the new multiline format then.

I was thinking of the future when these notifications would surface on the user interface, but I'll deal with that problem then.

Arguably, having to look at the logs for warnings and informational messages is a terrible, terrible user experience — all these warnings should be GUI notifications. While the logs can be useful for debugging for power users, regular users should never _have_ to look at the logs like ever in their lives, and should be able to use Staging effectively.

Reply:
Well, once the OSD is available, a lot more will have to change/improve, not only this little non-critical warning :)

Reply:
> Well, once the OSD is available, a lot more will have to change/improve, not only this little non-critical warning :)

Yeah, no doubt 😆 Hence I'm like, yeah, do whatever for now 😆